yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Can you delete this file? There should only be a yarn.lock within `wvc-ourwork-nextjs/`

Reply:
Ty!